.github/workflows/sycl-benchmark.yml

```diff
@@ -0,0 +1,215 @@
+name: Run benchmarks (using sycl-bench)
```

Comment:
I'd reword the workflow name. Something like `Run sycl-bench microbenchmarks`

Reply:
Fixed in #14454